interactors.js

```diff
@@ -1,6 +1,7 @@
 export * from './lib/SearchForm/tests/interactor';
 export * from './lib/SearchAndSortPane/tests/SearchAndSortInteractor';
 export * from './lib/FullScreenForm/tests/interactor';
+export { default as PreloaderInteractor } from './lib/Preloader/tests/interactor';
```

Comment:
I suggest following import convention described above but understand why you have used this syntax.

`export default interactor(class PreloaderInteractor {`
So for usages like this we can come up with
```
@interactor
export class PreloaderInteractor {
```

Reply:
@VictorSoroka1 it seems there were not agreements about the import/exporting process. So let's discuss and describe it to be on the same page in the future. There is a separated story to make them consistent after the final decision.